sstables/sstables.hh

```diff
@@ -880,7 +880,7 @@ public:
 
     // Gets ratio of droppable tombstone. A tombstone is considered droppable here
     // for cells expired before gc_before and regular tombstones older than gc_before.
```

Comment:
Comment needs update?

Reply:
gc_before is internalized, so we probably should move the second part inside and make the first part more descriptive, as it's only a description of function name.